src/tests/DomainHost.t.sol

```diff
@@ -515,11 +551,12 @@ contract DomainHostTest is DssTest {
         // Simulate user getting some gems for this ilk (normally handled by end)
         vat.slip(ILK, address(this), 50 ether);
 
+        uint256 claim = 50 ether * (host.debt() / host.grain());
```

Comment:
I would use hardcoded values here instead of the claim calculation to avoid mistakes in the comments like above.

Reply:
Done.